frappe/core/doctype/domain/domain.py

```diff
@@ -28,7 +28,7 @@ def setup_domain(self):
 		self.setup_properties()
 		self.set_values()
 
-		if not int(frappe.defaults.get_defaults().setup_complete or 0):
+		if not frappe.cint(frappe.defaults.get_defaults().setup_complete):
```

Comment:
Avoid using indirect imports like this. Just the way you removed unused import someday frappe.cint will also be removed :grin:

Reply:
Fixed